### PR TITLE
add DE-ForceFields

### PR DESCRIPTION
**Pull request by jthorton**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by djcole56** [comment]
Looks good to me

**Comment by pavankum** [comment]
Looks great!

**Comment by SimonBoothroyd** [comment]
I'd be happy to be a maintainer

**Comment by jthorton** [comment]
@conda-forge/help-python ready for review!

**Comment by jthorton** [comment]
@conda-forge-admin, please restart ci

**Comment by jthorton** [comment]
https://github.com/orgs/conda-forge/teams/help-python ready for review!

**Comment by jthorton** [comment]
@conda-forge-admin, please ping team

File: recipes/de-forcefields/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  sha256: 3f28755a9dfe3c6fdc02a9c55c89c7b511952aab27025567132fde0090ef46f9
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python >=3.7, <3.10
```
Comment (ocefpaf) [inline recipes/de-forcefields/meta.yaml:19]:
```suggestion
    - python >=3.7,<3.10
```

File: recipes/de-forcefields/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - pip
+  run:
+    - python >=3.7, <3.10
```
Comment (ocefpaf) [inline recipes/de-forcefields/meta.yaml:22]:
```suggestion
    - python >=3.7,<3.10
```

File: recipes/de-forcefields/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/jthorton/de-forcefields/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 3f28755a9dfe3c6fdc02a9c55c89c7b511952aab27025567132fde0090ef46f9
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (ocefpaf) [inline recipes/de-forcefields/meta.yaml:15]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

**Comment by jthorton** [comment]
Thanks very much for the help @ocefpaf!

**Pull request merged**